hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBBatchOperation.java

```diff
@@ -135,9 +135,9 @@ public String toString() {
     }
   }
 
-  private abstract class Operation implements Closeable {
+  private abstract static class Op implements Closeable {
 
-    private Operation() {
+    private Op() {
     }
```

Comment:
Remove this empty constructor.

Reply:
@swamirishi , this is not yet addressed.

Reply:
This would be useful in the next PR. I have removed it and I will add it in the next PR.